### ISSUE TITLE
expose the cumulus version via metric

### ISSUE THREAD

**Issue by wasabi222** [tynany/cumulus_exporter#6]
Would it be possible to expose the cumulus version via a metric? In node_exporter this is done by setting a guage to 1 with the relevant info like so: 
```
node_uname_info{domainname="(none)",machine="x86_64",nodename="dev1-cl",release="4.1.0-cl-7-amd64",sysname="Linux",version="#1 SMP Cumulus 4.1.33-1+cl3u20 (2019-04-26)"} 1
```

It would be nice to have something like
```
cumulus_version_info{major="4", minor="1", patch="1"} 1
```

or similar

**Comment by tynany** [comment]
Apologies for the delay. I have implemented the below metrics in version [0.1.2](https://github.com/tynany/cumulus_exporter/releases/tag/v0.1.2):

```
# HELP cumulus_version_info Cumulus version info.
# TYPE cumulus_version_info gauge
cumulus_version_info{major="4",minor="1",patch="1",release="4.1.1"} 1
# HELP cumulus_version_major Cumulus major release.
# TYPE cumulus_version_major gauge
cumulus_version_major 4
# HELP cumulus_version_minor Cumulus minor release.
# TYPE cumulus_version_minor gauge
cumulus_version_minor 1
# HELP cumulus_version_patch Cumulus patch release.
# TYPE cumulus_version_patch gauge
cumulus_version_patch 1
```

Does this cover your use case?

**Comment by tynany** [comment]
Closing due to no response.
